docs/advanced/theme/manifest-json.mdx

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 9
+title: manifest.json
+description:
+  This guide explains how to customize the web application manifest served by
+  Front-Commerce so that your website is recognized as a Progressive Web
+  Application (PWA).
+---
+
+<p>{frontMatter.description}</p>
+
+## Front-Commerce web application manifest generation
+
+Front-Commerce implements the following mechanism to generate [a web app
+manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) so that the web
+browser can detect that it's a Progressive Web Application (PWA):
+
+
+1. Pages generated by Front-Commerce include the following markup:
+  ```html
+  <link rel="manifest" href="http://your.website.example.com/manifest.json" />
+  ```
+  so that the browser knows where to find this web app manifest.
+1. The `manifest.json` is dynamically generated by Front-Commerce based on
+   `public/manifest.json` available on the disk and on the project
+   environment (URLs, …).
+
+
+## Customizations
+
+### `manifest.json`
+
+To customize the `manifest.json` generated by Front-Commerce, you have to copy
+the `public/manifest.json` from Front-Commerce to your project:
+
+```bash
+# from the root of project, usually a clone of the skeleton
+mkdir src/public
+cp node_modules/front-commerce/public/manifest.json src/public/manifest.json
+```
+
+Then you can customize `src/public/manifest.json` content to match your project.
+Commonly, you will change the `name`, `short_name`, the colors and the icons.
```

Comment:
We could link to https://realfavicongenerator.net/ as a tool to generate correct manifest with icons and colors?